oneflow/ir/lib/OneFlow/OneFlowOps.cpp

```diff
@@ -484,7 +484,11 @@ llvm::SmallVector<Value, 4> BiasAddOp::NchwToNhwc(llvm::SmallVector<Value, 4> va
   return results;
 }
 
-bool NormalizationOp::IsNCHW() { return this->axisAttr().getValue().getSExtValue() == 1; }
+bool NormalizationOp::IsNCHW() {
```

Comment:
fix oneflow/python/oneflow/test/graph/test_free_tensor_not_in_job.py bug when enable auto_nhwc in ci.